src/nifgen/metadata/enums_addon.py

```diff
@@ -8,4 +8,18 @@
 enums_codegen_method = {
 }
 
+enums_additional_enums = {
+    'RelativeTo': {
+        'values': [
+            {
+                'name': 'WAVEFORM_POSITION_START',
```

Comment:
I have a change I am working on that will remove common prefixes and suffix like `WAVEFORM_POSITION_` from enum names, if they all have it.

Since we are explicitly adding this enum, I think we should not have that common prefix. I.e. make these `START` and `CURRENT`.